src/coreclr-debug/activate.ts

```diff
@@ -40,15 +40,33 @@ export function activate(context: vscode.ExtensionContext, reporter: TelemetryRe
         return;
     }
 
+    let dotnetVersion: string = '';
+    let osVersion: string = '';
+    let osRID: string = '';
+    _util.spawnChildProcess('dotnet', ['--info'], _util.coreClrDebugDir(), (data: Buffer) => {
```

Comment:
>_util [](start = 4, length = 5)

The call on line 30 is just a call to 'dontet --version' to see if it dotnet can successfully launch. We should probably combine these calls together so we don't have the extra spawn for no reason.